library/globals.inc.php

```diff
@@ -3057,6 +3057,16 @@ function gblTimeZones()
             xl('Website link for the Patient Portal.')
         ),
 
+        'portal_css_header' => array(
+            xl('Portal Default Theme'),
+            array(
+                'style_light.css' => xl('Light'),
+                'style_dark.css' => xl('Dark')
+            ),
+            'style_light.css',
+            xl('Pick a default portal theme.')
+        ),
```

Comment:
Would be cool if also made this a patient specific setting. Probably wouldn't be to hard to make a mechanism (akin to the user specific settings) that would support that.

Reply:
yep I'll persist somewhere in portal prob on new landing page. The global is a default.